### ISSUE TITLE
New maintainer

### ISSUE THREAD

**Issue by Fraaxius** [PixelOS-AOSP/official_devices#397]
## Name and Codename of the device you want to apply for 
```
Motorola Edge 30 (dubai)
```

## Device Tree sources

```
https://github.com/Fraaxius/android_device_motorola_dubai
https://github.com/Fraaxius/android_device_motorola_sm7325-common
https://github.com/Fraaxius/kernel_motorola_sm7325
https://github.com/Fraaxius/vendor_motorola_dubai
https://github.com/Fraaxius/vendor_motorola_sm7325-common
```

## What ROM's do you maintain officially/unofficially
```
At the moment, i'm only maintaining PixelOS.
```

## How long have you been building Custom ROMs
```
From 4 years ago (since August 2020).
```

## Any Exceptions/special concern?
```
Within the time that I mentioned above, I have worked on some Motorola devices (Moto G7 Plus, Moto G9 Plus, Moto G7 Power). I had official ROMs like RevengeOS, The Styx Project and NezukoOS (A rom from 2021), on G7 Plus (lake), I have subsequently kept unofficial ROMs for a while on G9 Plus.
```

## List of changes/patches applied to source, if any
```
https://github.com/LineageOS/android_hardware_motorola (For MotoActions) and https://github.com/LineageOS/android_system_qcom (Dependency for hardware_motorola repository)
```

### Telegram username
```
t.me/Francirijillo
```

### Link to your XDA thread of unofficial build

```
https://xdaforums.com/t/rom-14-unofficial-pixelos-for-motorola-edge-30-dubai-aosp-stable.4683925/
```

### Link to your unofficial PixelOS Build Post other than XDA

```
First build: https://t.me/MotoEdge30Updates/160
Second build : https://t.me/MotoEdge30Updates/165
Third build (Actual update): https://t.me/MotoEdge30Updates/178
```

**Comment by Fraaxius** [comment]
Hi, I would appreciate it if you could give me a final verdict, please :). Thanks :D @whyredfire @kawaaii @cyberknight777 @basamaryan @bheatleyyy

**Comment by kawaaii** [comment]
https://github.com/Fraaxius/android_device_motorola_sm7325-common/commit/7a6adea50d6bbbcf5471168b65ad363a6c3a1d69
Unrequired

https://github.com/Fraaxius/android_device_motorola_sm7325-common/commit/b2609af41c584959898e01f0500420c295233ecc
If you're not going to be doing any update on those files, then move them to vendor.

Squash commits 6626f88c26f2430afba97b9bc7f01e30921a7de4, 702db65ddd1013d6aa95d979061bb95f1df259a9, d211a0155e37e6724d83ecaea3516e2d2658d9bc, and f0d297570daa6a04569827b0bddee4c97bd826c8 into fb6219b6a907008ec0a2ce0728e746d2bb174ca3.

Do similarly for 546e545d518f18b7bd9a25882a6eb013ad6fdf34, d450fce5c893432235d4e8c8a6f3505b6f61f93b and f40529df02c847e4834e494a91ed7151ff315740.

**Comment by Fraaxius** [comment]
- **Squashed all commits from your request, all on** https://github.com/Fraaxius/android_device_motorola_dubai/commit/71af0fdb04529684469a8d73fea8a45d3f97a2d6

- **Dropped unnecessary commit** https://github.com/Fraaxius/android_device_motorola_sm7325-common/commit/7a6adea50d6bbbcf5471168b65ad363a6c3a1d69

- **Moved Dolby configs to** https://github.com/Fraaxius/vendor_motorola_sm7325-common/commit/683426ef41934bebb41ecb9ba29b7939ead052b1

_Is necessary move Dolby VINTFs and Sepolicies to vendor too? I kept them in_ https://github.com/Fraaxius/android_device_motorola_sm7325-common/commit/bc07b4501017f82f85bfd4d3344a76cba394238a

**Comment by kawaaii** [comment]
> * **Squashed all commits from your request, all on** [Fraaxius/android_device_motorola_dubai@71af0fd](https://github.com/Fraaxius/android_device_motorola_dubai/commit/71af0fdb04529684469a8d73fea8a45d3f97a2d6)
> * **Dropped unnecessary commit** [Fraaxius/android_device_motorola_sm7325-common@7a6adea](https://github.com/Fraaxius/android_device_motorola_sm7325-common/commit/7a6adea50d6bbbcf5471168b65ad363a6c3a1d69)
> * **Moved Dolby configs to** [Fraaxius/vendor_motorola_sm7325-common@683426e](https://github.com/Fraaxius/vendor_motorola_sm7325-common/commit/683426ef41934bebb41ecb9ba29b7939ead052b1)
> 
> _Is necessary move Dolby VINTFs and Sepolicies to vendor too? I kept them in_ [Fraaxius/android_device_motorola_sm7325-common@bc07b45](https://github.com/Fraaxius/android_device_motorola_sm7325-common/commit/bc07b4501017f82f85bfd4d3344a76cba394238a)
> 
> @kawaaii

Just vintfs to vendor is fine.

**Comment by Fraaxius** [comment]
:D that's right

**Comment by Fraaxius** [comment]
@kawaaii i have a question, both vintfs (configs/vintf/framework_compatibility_matrix_dolby.xml and configs/vintf/manifest_dolby.xml) needs to move to vendor?

**Comment by Fraaxius** [comment]
- Moved manifest_dolby to https://github.com/Fraaxius/vendor_motorola_sm7325-common/commit/9b3cacb645d3c2976b5d159abf37c0919f185732#diff-d4f7da0483654a7b41f1a51c38e269207bd43dadc225dccab6833eabf7ec1805

- Moved Dolby's framework compabillity to https://github.com/Fraaxius/android_device_motorola_sm7325-common/commit/c1e40b48a5ff71d147f883384f9ac0702e74b3b5#diff-ef3c449fbb07d3617d8f567b9b7de911a439e264827060956b0e9ab35399c7d9

@kawaaii

**Comment by kawaaii** [comment]
What happened to your dolby changes in proprietary files list?

**Comment by Fraaxius** [comment]
> What happened to your dolby changes in proprietary files list?

I have removed them for the meantime, because Stock Moto has some similar blobs to the Dolby ones that I imported from OnePlus 9R. Also, due to the modification I will have to update the blobs of the latest firmware they released

**Comment by Fraaxius** [comment]
It's temporary, I will import them again when I have already updated the blobs in the common vendor, to avoid all kinds of conflicts

**Comment by kawaaii** [comment]
Fine

**Comment by whyredfire** [comment]
Hi, any specific reason you reset this tree and re-picked the same changes?

https://github.com/LineageOS/android_device_motorola_sm7325-common

**Comment by Fraaxius** [comment]
> Hi, any specific reason you reset this tree and re-picked the same changes?
> 
> https://github.com/LineageOS/android_device_motorola_sm7325-common

Just for keep updated from LineageOS trees.

**Comment by whyredfire** [comment]
> > Hi, any specific reason you reset this tree and re-picked the same changes?
> > https://github.com/LineageOS/android_device_motorola_sm7325-common
> 
> Just for keep updated from LineageOS trees.

I don't think you understand what I wrote.

**Comment by Fraaxius** [comment]
> > > Hi, any specific reason you reset this tree and re-picked the same changes?
> > > https://github.com/LineageOS/android_device_motorola_sm7325-common
> > 
> > 
> > Just for keep updated from LineageOS trees.
> 
> I don't think you understand what I wrote.

I will try to explain as best as possible: To do the bringup, I used the LineageOS trees to do the rebase (also because it is the most stable tree that exists), I made the required changes for PixelOS, and I have updated them according to the latest changes from LOS. Together with changes to fix some problems (like the hotspot in 802.11ax or Wifi6), etc. I hope it has been understood as best as possible.
